### PR TITLE
Feat : Design services section of mobile app development page

### PR DESCRIPTION
**Pull request by cp-jagruti-a**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [x] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI 
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook  (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices  (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

**UI Video :**

https://github.com/canopas/canopas-website/assets/85920585/9e4e1f50-9fbd-4b22-9d0b-bfe6af2c90cc

**Performance:**
**Desktop:**
Screenshot from 2023-06-23 12-32-37
**Mobile**
Screenshot from 2023-06-23 12-33-14

### CODE REVIEW

File: vue-frontend/src/components/mobile-app-development/Services.vue [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+import android from "@/assets/images/mobile-app-development/services/android/android.webp";
+import gradle from "@/assets/images/mobile-app-development/services/android/gradle.webp";
+import kotlin from "@/assets/images/mobile-app-development/services/android/kotlin.webp";
+import java from "@/assets/images/mobile-app-development/services/android/java.webp";
+import clang from "@/assets/images/mobile-app-development/services/android/C++.webp";
+import ios from "@/assets/images/mobile-app-development/services/ios/iOS.webp";
+import swift from "@/assets/images/mobile-app-development/services/ios/swift.webp";
+import javaScript from "@/assets/images/mobile-app-development/services/ios/javaScript.webp";
+import objectiveC from "@/assets/images/mobile-app-development/services/ios/objectiveC.webp";
+import Csharp from "@/assets/images/mobile-app-development/services/ios/Csharp.webp";
```
Comment (cp-priyanka-g) [inline vue-frontend/src/components/mobile-app-development/Services.vue:121]: keep the naming convention consistent as other names.

File: vue-frontend/src/components/mobile-app-development/Services.vue [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+            "Bring your multi-platform app vision to life with our Flutter app development services. We excel in crafting high-quality, stunning applications that work flawlessly on Android, iOS, and the web, all from a single codebase, facilitating faster time to market and cost savings.",
+        },
+      ],
+    };
+  },
+};
+</script>
+<style scoped>
+.gradient-border {
+  border-image: linear-gradient(to bottom, #f2709c, #ff835b) 1;
```
Comment (cp-priyanka-g) [inline vue-frontend/src/components/mobile-app-development/Services.vue:199]: you can add this class to tailwind.config and use it.

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/mobile-app-development/Services.vue (hunk 2 above)
Comment (cp-jagruti-a) [reply vue-frontend/src/components/mobile-app-development/Services.vue:199]: we can add background image like that not border image tailwind is not providing way to do it

File: vue-frontend/src/components/home-new/ServiceSection.vue [hunk 3]
```diff
@@ -93,5 +95,10 @@ export default {
       ],
     };
   },
+  methods: {
+    openUrl(url) {
+      window.open(url, "_self");
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ServiceSection.vue:100]: Will it open in live also?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/home-new/ServiceSection.vue [hunk 4]
```diff
@@ -16,30 +16,31 @@
...
       @touchstart.passive="activeIndex = index"
+      @click="!showServices ? openUrl(service.url) : ''"
       :class="
         activeIndex === index
-          ? 'hover:tw-text-transparent tw-transition-all tw-duration-500  tw-ease-in-out tw-transform'
+          ? 'tw-transform tw-transition-all tw-duration-500  tw-ease-in-out hover:tw-text-transparent'
           : ''
       "
-      class="tw-relative tw-flex tw-items-center tw-h-36 tw-pl-16 xl:tw-pl-48 xll:tw-pl-[40rem]"
+      class="tw-relative tw-flex tw-h-36 tw-cursor-pointer tw-items-center tw-pl-16 xl:tw-pl-48 xll:tw-pl-[40rem]"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ServiceSection.vue:25]: If div is clickable, then only cursor-pointer should be there

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Review by cp-priyanka-g — DISMISSED** [review]

File: vue-frontend/src/components/home-new/ServiceSection.vue [hunk 5]
```diff
@@ -16,30 +16,32 @@
...
+      @click="!showServices ? openUrl(service.url) : ''"
+      :class="[
         activeIndex === index
-          ? 'hover:tw-text-transparent tw-transition-all tw-duration-500  tw-ease-in-out tw-transform'
-          : ''
-      "
-      class="tw-relative tw-flex tw-items-center tw-h-36 tw-pl-16 xl:tw-pl-48 xll:tw-pl-[40rem]"
+          ? 'tw-transform tw-transition-all tw-duration-500  tw-ease-in-out hover:tw-text-transparent'
+          : '',
+        !showServices && index == 0 ? 'tw-cursor-pointer' : '',
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ServiceSection.vue:24]: Why is index == 0 taken?

File: vue-frontend/src/components/home-new/ServiceSection.vue [hunk 6]
```diff
@@ -60,15 +62,18 @@
 </template>
 
 <script type="module">
+import config from "@/config.js";
 export default {
   data() {
     return {
       activeIndex: null,
+      showServices: config.IS_PROD,
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ServiceSection.vue:70]: This is confusing.
You can use it like `showServices: !config.IS_PROD`

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Review by cp-priyanka-g — APPROVED** [review]

**Pull request merged**